command_plugins/github/plugin.py

```diff
@@ -48,6 +48,14 @@ def __init__(self):
                 "permitted_permissions": ["push", "pull"],  # To grant admin, add this to the config for
                 "enabled": True  # this command in the config.py.
             },
+            "!SetRepoTeamPermissions": {
```

Comment:
Please rename to `!SetRepoPermissions`.

Reply:
The logic behind the naming was that `set_repo_permissions` is an existing funciton, and I didn't want the command to invoke it directly. I'm gonna change as requested, but since it invokes `set_team_permissions` perhaps some more thought should be put into this.

Reply:
For the command functions, I typically suffix them with `command`. However, in this case, this is the actual command that the user will type into Slack.  Would prefer a shorter command name.